### PR TITLE
sio03 added

### PR DESCRIPTION
**Pull request by MBeer2605**
added SIO03, modified SIO02

### CODE REVIEW

File: docs/user-docs/_lyve/lyve-tracelets.md
Comment (KlausPopp) [inline docs/user-docs/_lyve/lyve-tracelets.md:1]: Remove (coming soon) from SIO03

File: docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+|                                        | GPS/QZSS (L1C/A L2C) |
+|                                        | GLONASS (L1OF L2OF) |
+|                                        | Galileo (E1B/C E5b) |
+|                                        | BeiDou (B1I B2I) |
+| Position accuracy                      | localization values have an accuracy of < 1m for 95% of the reported values |
+| Speed Pulse Signal                     | acc. to IEC 16844-2 (input high: 4,8V; input low: 2,2V)|
+| Ignition                               | On State: Input high: 5,2 V (min) or open |
+|                                        | Standby State (after delay ~3 sec): Input low: 3,6 V (max) |
+| **Maintenance**                        |
+| Firmware update                        | Via USB, WLAN |
```
Comment (KlausPopp) [inline docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md:58]: via LAN

File: docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+| Management                             | Via io4edge protocol, see https://docs.ci4rail.com/edge-solutions/io4edge/ |
```
Comment (KlausPopp) [inline docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md:59]: Use link [io4edge protocol]({{ '/edge-solutions/io4edge' | relative_url }})

File: docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+# Connections
+
+SIO03 provides two M12 interface connectors for roboust and IP protected connections.
+The product offers an M12-8pin A-coded connector for shared power and service interfaces as well as an M12-4pin D-coded Ethernet interface connector.
+
+## M12-8pin A-coded, socket:
+* alt. power supply 12V / 24V DC (nom)
+* Ignition
+* Wheeltic Input
```
Comment (KlausPopp) [inline docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md:94]: Wheeltick

File: docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+
+# Mounting & Installation
+
+The tracelet has to be mounted with clear view to GNSS satellites as well as UWB satlet infrastructure.
+We recommend mounting on roof of vehicle.
+
+Mounting depends on the given conditions per vehicle. Depending on the mounting conditions, specific mounting brackets might be available. Contact Ci4Rail for specific designs of mounting brackets
+
+![Lyve Tracelet]({{ '/user-docs/images/lyve/SIO02_model_halterung.png' | relative_url }})
```
Comment (KlausPopp) [inline docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md:152]: Drawings show SIO02

**Review by KlausPopp — CHANGES_REQUESTED** [review]

File: docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md (hunk 4 above)
Comment (MBeer2605) [reply docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md:152]: will do a comment, that equal for SIO02 and SIO03 - no specific drawings will be done

File: docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md (hunk 1 above)
Comment (MBeer2605) [reply docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md:58]: done

File: docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md (hunk 2 above)
Comment (MBeer2605) [reply docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md:59]: done

File: docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md (hunk 1 above)
Comment (MBeer2605) [reply docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md:58]: done

File: docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md (hunk 2 above)
Comment (MBeer2605) [reply docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md:59]: done

File: docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md (hunk 3 above)
Comment (MBeer2605) [reply docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md:94]: done

Comment (MBeer2605) [reply docs/user-docs/_lyve/lyve-tracelets/sio03/detailed-description.md:94]: done

File: docs/user-docs/_lyve/lyve-tracelets.md
Comment (MBeer2605) [reply docs/user-docs/_lyve/lyve-tracelets.md:1]: done

**Review by KlausPopp — APPROVED** [review]

**Comment by MBeer2605** [comment]
SIO03 Branch merged into main

**Pull request merged**